plugins/chat/app/services/chat/lookup_channel_threads.rb

```diff
@@ -180,5 +201,14 @@ def tracked_threads_subquery(guardian, channel)
         )
         .to_sql
     end
+
+    def build_load_more_url(contract:, **)
+      load_more_params = { offset: contract.offset + contract.limit }.to_query
+      context.load_more_url =
```

Comment:
Was going to suggest the rails route helpers for this to make it a bit more resilient, but although it has the API channel path it doesn't generate the paths for threads for some reason :S 

```ruby
Chat::Engine.routes.url_helpers.api_channels_path
=> "/chat/api/channels"
```

```ruby
 Chat::Engine.routes.url_helpers.methods.filter {|m| m.to_s.include?("api")}
=> [:api_chatables_path,
 :api_channels_path,
 :api_channels_me_path,
 :api_channels_read_path,
 :api_path,
 :api_mentions_groups_path,
 :api_direct_message_channels_path,
 :api_url,
 :api_chatables_url,
 :api_channels_url,
 :api_channels_me_url,
 :api_channels_read_url,
 :api_mentions_groups_url,
 :api_direct_message_channels_url,
 :user_api_key_new_path,
 :user_api_key_path,
 :user_api_key_revoke_path,
 :user_api_key_undo_revoke_path,
 :user_api_key_otp_path,
 :web_hook_events_admin_api_index_path,
 :admin_api_index_path,
 :user_api_key_new_url,
 :user_api_key_url,
 :user_api_key_revoke_url,
 :user_api_key_undo_revoke_url,
 :user_api_key_otp_url,
 :web_hook_events_admin_api_index_url,
 :admin_api_index_url]
```

Not a big deal, just confusing behaviour from rails.

Reply:
It also obviously recognizes these routes:

```
bundle exec rails routes | rg threads
                            GET    /api/channels/:channel_id/threads(.:format)                                      chat/api/channel_threads#index {:format=>:json}
                            PUT    /api/channels/:channel_id/threads/:thread_id(.:format)                           chat/api/channel_threads#update {:format=>:json}
                            GET    /api/channels/:channel_id/threads/:thread_id(.:format)                           chat/api/channel_threads#show {:format=>:json}
                            PUT    /api/channels/:channel_id/threads/:thread_id/read(.:format)                      chat/api/thread_reads#update {:format=>:json}
                            PUT    /api/channels/:channel_id/threads/:thread_id/notifications-settings/me(.:format) chat/api/channel_threads_current_user_notifications_settings#update {:format=>:json}
```